Infusion/Direction.cs

```diff
@@ -12,6 +12,7 @@ public struct Direction
         public static readonly Direction Southwest = (Direction) 0x05;
         public static readonly Direction West = (Direction) 0x06;
         public static readonly Direction Northwest = (Direction) 0x07;
+        public static readonly Direction Mask = (Direction)0x07;
```

Comment:
Why `Mask`? It is unused and `Direction` doesn't hold the speed. Infusion decomposes them to `Direction` itself and `MovementType`.